just-not-sorry.css

```
@@ -4,7 +4,5 @@
 
 .jns-tooltip {
   max-width: 380px;
-  max-height: 100px;
-  margin-bottom: 30px;
-  margin-top: 50px;
+  border: solid white;
```

The border is a little bold.  How about this instead?
```suggestion
  border: 1px solid darkgray;
```